src/Compilers/VisualBasic/Test/IOperation/IOperation/IOperationTests_IObjectCreationExpression.vb

```diff
@@ -1252,6 +1252,33 @@ IObjectCreationOperation (Constructor: Sub S..ctor()) (OperationKind.ObjectCreat
             VerifyOperationTreeAndDiagnosticsForTest(Of ObjectCreationExpressionSyntax)(source, expectedOperationTree, expectedDiagnostics)
         End Sub
 
+        <CompilerTrait(CompilerFeature.IOperation)>
+        <Fact()>
+        Public Sub ObjectCreationCollectionWithInaccessibleStructureConstructor()
+            Dim source = <![CDATA[
+Class C
+    Protected Structure S
+    End Structure
+End Class
+Class D
+    Shared Sub M(o)
+        M(New C.S())'BIND:"New C.S()"
+    End Sub
+End Class]]>.Value
+            Dim expectedDiagnostics = <![CDATA[
```

Comment:
>Dim expectedDiagnostics = <![CDATA[ [](http://example.com/codeflow?start=12&length=35)

Consider adding empty lines between multiline statements, this is going to improve readability.